test/Cnblogs.Architecture.IntegrationTestProject/Program.cs

```diff
@@ -22,11 +29,12 @@
     app.UseSwaggerUI();
 }
 
-app.UseHttpsRedirection();
-
 app.UseAuthorization();
 
 app.MapControllers();
+
+app.Subscribe<TestIntegrationEvent>(appName);
```

Comment:
```suggestion
app.Subscribe<TestIntegrationEvent>();
```

`Subscribe()` 会自动从 AssemblyInfo 里拿到被订阅事件的 appName，不需要手动指定。